services/Skyra.Database/Models/Banner.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+#nullable disable
+
+namespace Skyra.Database.Models
+{
+	[Table("banner")]
+	public partial class Banner
```

Comment:
any reason why these are partial classes?

Reply:
Ask EFCore, not me.

Reply:
does rider find any other partial classes for any of these? if not, it'll be a good idea to make them sealed, not partial

Reply:
That wouldn't be wise, in fact, there are keys that are `virtual` (EFCore's relationships).

I ran Rider's code cleanup on the whole solution, it removed the partials.

Reply:
perfect!